src/CloudFront/Policy.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Aws\CloudFront;
+
+class Policy
+{
+    private $resource;
+    private $until;
+    private $from;
+    private $ip;
+
+    /**
+     * Creates a CloudFront policy for allowing access to restricted
+     * distributions.
+     *
+     * @param string            $resource   The resource (URL or RTMP path) to
+     *                                      which this policy grants access.
+     * @param string|int        $until      UTC Unix timestamp indicating when
+     *                                      this policy will expire.
+     * @param string|int|null   $from       (Optional) UTC Unix timestamp
+     *                                      indicating when this policy will
+     *                                      become valid.
+     * @param string|null       $ip         (Optional) The IP address range
+     *                                      (expressed in standard IPv4 CIDR
+     *                                      format) of the client(s) permitted
+     *                                      to make requests under this policy.
+     *
+     * @link http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-setting-signed-cookie-custom-policy.html#private-content-custom-policy-statement-cookies
+     */
+    public function __construct($resource, $until, $from = null, $ip = null)
```

Comment:
@mtdowling That's a good point. I could simplify the class by having it only support canned policies (resource + expiration time), as that's a much stronger contract than CloudFront's current policy spec.